mlflow/server/js/.eslintrc.js

```diff
@@ -359,7 +361,7 @@ module.exports = {
     'valid-typeof': 2,
     'vars-on-top': 0,
     'wrap-iife': 2,
-    'wrap-regex': 2,
+    'wrap-regex': 0,
```

Comment:
Prettier removes `()` here, which conflicts with [wrap-regex](https://eslint.org/docs/rules/wrap-regex).

<img width="557" alt="Screen Shot 2020-04-26 at 17 32 13" src="https://user-images.githubusercontent.com/17039389/80302290-0f776e80-87e4-11ea-8930-b5dacab76d82.png">

Reply:
This lint rule makes sense. Let's keep it by removing the conflicting prettier rule.